dts/arm/armv8-m.dtsi

```diff
@@ -0,0 +1,25 @@
+#include "skeleton.dtsi"
+
+/ {
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "simple-bus";
+		interrupt-parent = <&nvic>;
+		ranges;
+
+		nvic: interrupt-controller@e000e100  {
+			compatible = "arm,v8m-nvic";
+			reg = <0xe000e100 0xc00>;
```

Comment:
Need a binding to match arm,v8m-nvic (dts/bindings/interrupt-controller/arm,v8m-nvic.yaml)

Reply:
Fixed, thanks! Added it to the second patch (namely, for Cortex-M33, consistent with the other .yaml bindings for CM0+ and CM4, respectively)